Resources/views/CRUD/base_show.html.twig

```diff
@@ -18,42 +20,42 @@ file that was distributed with this source code.
 {% block tab_menu %}{{ knp_menu_render(admin.sidemenu(action), {'currentClass' : 'active', 'template': sonata_admin.adminPool.getTemplate('tab_menu_template')}, 'twig') }}{% endblock %}
 
 {% block show %}
-    <div class="sonata-ba-view row">
-
-        {{ sonata_block_render_event('sonata.admin.show.top', { 'admin': admin, 'object': object }) }}
-
-        {% for name, view_group in admin.showgroups %}
-            <div class="{{ view_group.class | default('col-md-12') }}">
-                <div class="{{ view_group.box_class }}">
-                    {% if name %}
-                        <div class="box-header with-border">
-                            <h4 class="box-title">
-                                {% block show_title %}
-                                    {{ admin.trans(name, {}, view_group.translation_domain) }}
-                                {% endblock %}
-                            </h4>
+    {{ sonata_block_render_event('sonata.admin.show.top', { 'admin': admin, 'object': object }) }}
+
+    {% set has_tab = (admin.showtabs|length == 1 and admin.showtabs|keys[0] != 'default') or admin.showtabs|length > 1 %}
+
+    {% if has_tab %}
+        <div class="nav-tabs-custom">
+            <ul class="nav nav-tabs" role="tablist">
+                {% for name, show_tab in admin.showtabs %}
+                    <li{% if loop.index == 1 %} class="active"{% endif %}>
+                        <a href="#tab_{{ admin.uniqid }}_{{ loop.index }}" data-toggle="tab">
+                            <i class="fa fa-exclamation-circle has-errors hide"></i> {{ admin.trans(name, {}, show_tab.translation_domain) }}
```

Comment:
Try to get rid of the horizontal scroll on github by linebreaking this line and others.